components/display/index.rst

```diff
@@ -26,6 +26,54 @@ using an API that is designed to
 
 .. _display-engine:
 
+Color test patron
+-----------------
+
+When you have issues with your color display you can best test your display with bellow test script.
+It will show 3 color bars in **RED**, **GREEN** and **BLUE**. To help the graphics display team determine 
+what the best way is to help you, we would love to see an image of the below result.
+
+Also when you create an issue about your display, please add the link where you brought the display from 
+so we can validate the used configuration.
```

Comment:
```suggestion
Should you create an issue in GitHub regarding your display, please include a link to where you purchased it 
so that we can validate the configuration you've used.
```